Utility/Utility.Public/Include/Utility/Utility.hpp

```diff
@@ -137,9 +137,7 @@ inline std::string removeSpaces(const std::string& input)
 
     while (sequenceStart < input.end())
     {
-        bool spaceSequence = isSpace(*sequenceStart);
-
-        if (spaceSequence)
+        if (bool spaceSequence = isSpace(*sequenceStart))
```

Comment:
Just notice: If you don't intend to use variables in the future, you don't need them.